backend/scripts/cleanup_lookups.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+from typing import Dict, Mapping
+
+from app.db.models import DocumentInstrument, DocumentKeyword, Instrument, Keyword
+from app.db.session import Base, SessionLocal
+
+
+LookupID = Mapping[str, int]
+
+
+def get_model_values(db: SessionLocal, table: Base) -> LookupID:  # type: ignore
+    id_lookup: Dict[str, int] = {}
+
+    for row in db.query(table).all():
+        should_update_lookup = row.name not in id_lookup or id_lookup[row.name] > int(
```

Comment:
Does this assume that ids are ascending when added to the table?

Reply:
well, it really doesn't matter, this was just because the original load had ascending IDs and I thought it was aesthetically pleasing to keep the lowest id... I guess I could have just used the first one I found & got exactly the same functionality.